### PR TITLE
Demo UI Containers

### PR DESCRIPTION
**Pull request by bbengfort**
This PR creates containers for the Demo UI and BFF -- updating them to production readiness using gunicorn and nginx.

### CODE REVIEW

File: containers/demo/bff/Dockerfile [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
+COPY web/demo/bff/flaskr .
+
+# Setup the environment
 ENV FLASK_APP flaskr
 ENV FLASK_ENV development
 
-COPY flaskr/ /flaskr
-
+# Initialize the database
 RUN [ "flask", "init-db" ]
```
Comment (bbengfort) [inline containers/demo/bff/Dockerfile:25]: @ayyoitsp This command is failing, am I missing a step?

**Comment by bbengfort** [comment]
Still TODO: manifest for k8s; but this should get local docker-compose running.

File: containers/demo/bff/Dockerfile (hunk 1 above)
Comment (ayyoitsp) [reply containers/demo/bff/Dockerfile:25]: I'll take a look tomorrow.

**Comment by bbengfort** [comment]
@ayyoitsp thanks for making the fix in #22 - I'll go ahead and squash and merge this.

**Pull request merged**